### PR TITLE
Fix rhosak UI test case

### PR DESCRIPTION
**Pull request by bdattoma**
This PR wants to fix the rhosak integration test which is broken

Signed-off-by: bdattoma [bdattoma@redhat.com](mailto:bdattoma@redhat.com)

### CODE REVIEW

**Comment by bdattoma** [comment]
[Jenkins Job](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1373/console) RUNNING

File: tests/Resources/Page/HybridCloudConsole/Rhosak.robot [hunk 1]
```diff
@@ -36,6 +36,7 @@ Create Kafka Stream Instance
     Click Element    xpath=//div[text()='${cloud_provider}']
     Click Element    id:form-cloud-region-option
     Click Element    xpath=//li/*[text()="${stream_region}"]
+    Sleep  2s
```
Comment (jgarciao) [inline tests/Resources/Page/HybridCloudConsole/Rhosak.robot:39]: Any chance this Sleep could be replaced with `Wait Until ...`

**Review by jgarciao — DISMISSED** [review]
LGTM. Added a suggestion. Can you test it again with Jenkins?

**Comment by bdattoma** [comment]
> LGTM. Added a suggestion. Can you test it again with Jenkins?

yes, it's in today plans

**Comment by bdattoma** [comment]
closing in favor of #627

**Pull request closed without merging**